app.js

```diff
@@ -151,8 +152,8 @@ var processRequest = function (req, res, isTar) {
       Orchestrator.hasFinished,
       function (state) {
         requests[id].results = state;
-        console.log(JSON.parse(JSON.stringify(requests[id])));
-        console.log('----------');
+        // console.log(JSON.parse(JSON.stringify(requests[id])));
+        // console.log('----------');
```

Comment:
These are remains of my test&hellip; I think the output in the console is too verbose.
Shouldn't we remove a few `console.log()` now?

Reply:
The `console.log` are commented out. You prefer to remove them completely?

Reply:
Done @tripu